src/makers/cotx/index.ts

```diff
@@ -0,0 +1,9 @@
+import antennas from './antennas'
+import hotspots from './hotspots'
+
+export default {
+  antennas,
+  hotspots,
+  id: 13,
```

Comment:
@cokes518 I dont see this in the onboarding server yet, has it been approved?

Reply:
> @cokes518 I dont see this in the onboarding server yet, has it been approved?

This id number is added by us according to the previously approved company id sequence. I am not sure how much id is written. Does the id number need to be approved by the onboarding server before it can be written here?